node/grpc/utils.go

```diff
@@ -88,6 +89,13 @@ func GetBlobHeaderFromProto(h *pb.BlobHeader) (*core.BlobHeader, error) {
 
 	quorumHeaders := make([]*core.BlobQuorumInfo, len(h.GetQuorumHeaders()))
 	for i, header := range h.GetQuorumHeaders() {
+		if header.GetQuorumId() > core.MaxQuorumID {
```

Comment:
This validation is redundant as we validated it as soon as the request entered the gRPC server (in server.go). But maybe fine since it's low cost.

Reply:
I think I noted at least one other place where this function was called without validation!